PetGame/Pages/Race.cshtml

```diff
@@ -0,0 +1,123 @@
+﻿@page
+@using PetGame.Pages
+@model RaceModel
+@{
+    Layout = "~/Pages/_Layout.cshtml";
+    ViewBag.Title = "Race";
+}
+<html lang="en">
+<h1>
+    Race Page
+</h1>
+<h2>
+    Spam button race!
+</h2>
+
+<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+<style>
+    canvas {
+        border: 1px solid #d3d3d3;
+        background-color: #f1f1f1;
+    }
+</style>
+<body onload="startGame()">
+    <script>
+var myGamePiece;
+
+function startGame() {
+    myGamePiece = new component(20, 20, "red", 10, 120);
+    myGameArea.start();
+}
+
+var myGameArea = {
+    canvas : document.createElement("canvas"),
```

Comment:
We should avoid creating elements, and instead have them already contained in the html body on load.